openedx/core/djangoapps/learner_pathway/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
shouldn't tests be in a separate directory ...

Reply:
done